compiler/tflite2circle/src/BuildBuiltinOptions/NonMaxSuppressionV4Options.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "NonMaxSuppressionV4Options.h"
+
+namespace tflite2circle
+{
+
+flatbuffers::Offset<circle::NonMaxSuppressionV4Options>
+build_circle_NonMaxSuppressionV4Options(flatbuffers::FlatBufferBuilder &fb,
+                                        const tflite::Operator *op)
+{
+  auto tflite_builtin_options = op->builtin_options_as_NonMaxSuppressionV4Options();
+  assert(tflite_builtin_options);
+  circle::NonMaxSuppressionV4OptionsBuilder builtin_options_builder{fb};
+  return builtin_options_builder.Finish();
+}
```

Comment:
(trivial)
```suggestion
build_circle_NonMaxSuppressionV4Options(flatbuffers::FlatBufferBuilder &fb,
                                        const tflite::Operator *)
{
  circle::NonMaxSuppressionV4OptionsBuilder builtin_options_builder{fb};
  return builtin_options_builder.Finish();
}
```

Reply:
Thanks! Probably left it this way before found out there should be no options in tflite's version of NonMaxSuppression.